### PR TITLE
chore: made all Options in commands private

### LINKED ISSUE #1058: Make all `Options` in commands private
The Options type in every command is public. This was originally done at the beginning with the anticipation is being used in tests, but since there is no need anymore, we can make all options private now.

### PR DESCRIPTION
**Pull request by alexal**
All of them are private now, except pkg/status/status.go.

Closes #1058

### CODE REVIEW

File: pkg/cmd/cluster/bind/bind.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func NewBindCommand(f *factory.Factory) *cobra.Command {
 	return cmd
 }
 
-func runBind(opts *Options) error {
+func runBind(opts *args) error {
```
Comment (craicoverflow) [inline pkg/cmd/cluster/bind/bind.go:76]: I know there was a discussion around the new name but I don't think we fully agreed on it (myself and @wtrocki coming from different view points). I think the opts variable name could be confusing with an `args` type. On consideration I think it is best to leave it as `options` for now, we can always revisit later but it causes the least disruption.

**Review by craicoverflow — COMMENTED** [review]
See comment re: leaving it as `options` for the time being.

**Comment by alexal** [comment]
> See comment re: leaving it as `options` for the time being.

Done.

**Review by craicoverflow — APPROVED** [review]
Thank you! 👍🏻 💯

**Pull request merged**